js/scripts.js

```diff
@@ -0,0 +1,16 @@
+var anyWord = prompt('Podaj dowolne słowo, na które mam wymienić te Papugi');
+var text = 'Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach';
+var splitResult = text.split (" ", 1);
+var wordUpperCased = anyWord.toUpperCase();
+var replacedWord = text.replace(splitResult, wordUpperCased);
+function getTextLenght(replacedWord) {
+	return replacedWord.lenght
+};
+var sentenceLenght = getTextLenght();
+function getslicingPoint (sentenceLenght) {
```

Comment:
ten `sentenceLenght` to taki nasz `x` - możesz go tak nazwać jak nazwałeś ale pamiętaj, że to nie ma znaczenia. Ma znaczenie to co potem przy wywołaniu `getslicingPoint` przekażesz - niestety Ty przekazujesz w 13 linijce `()` czyli ten nasz "x" będzie w funkcji undefined.

Ten sam błąd robisz przy funkcji `getTextLenght` - obiecujesz mu w 6 linijce, że dostanie jedną rzecz, a jej potem w 9 linijce przy wywołaniu nie dostaje.

Swoją drogą w 7 linijce też jest błąd, nie ma czegoś takiego jak `lenght` - jest `length`